.drone.yml

```diff
@@ -16,14 +16,13 @@ pipeline:
     image: node:slim
     environment:
       - DEPS_DIR=/usr/app/deps
-      - FRONT_BUILD_DIR=/usr/app/front/build
     commands:
       - npm install
       - npm install -g gulp-cli
       - gulp build
     volumes:
       - /tmp/cache/drone/stroyprombeton/site-packages/${DRONE_COMMIT_SHA}/site-packages:/usr/app/deps
-      - /tmp/cache/drone/stroyprombeton/front_build/${DRONE_COMMIT_SHA}:/usr/app/front/build
+      - /tmp/cache/drone/stroyprombeton/front_build/${DRONE_COMMIT_SHA}:/drone/src/github.com/fidals/stroyprombeton/build
```

Comment:
seems we can just remove this strange mapping.
Or at least not touch it

Reply:
@duker33 It is required to mount built static files to other pipes

Reply:
let's continue discussion in this task:
https://github.com/fidals/refarm-site/issues/132